extensionsGallery-insider.json

```diff
@@ -909,6 +909,63 @@
 									"value": "*"
 								}
 							]
+						},
+						{
+							"extensionId": "27",
+							"extensionName": "newdatabase",
+							"displayName": "New Database",
+							"shortDescription": "Adds a 'New Database' command. Right-click on the Databases for a connection in Object Explorer, or run 'New Database' from the command palette.",
+							"publisher": {
+								"displayName": "Kevin Cunnane",
+								"publisherId": "kevcunnane",
+								"publisherName": "kevcunnane"
+							},
+							"versions": [
+								{
+									"version": "0.0.3",
+									"lastUpdated": "11/19/2018",
+									"assetUri": "",
+									"fallbackAssetUri": "fallbackAssetUri",
+									"files": [
+										{
+											"assetType": "Microsoft.SQLOps.DownloadPage",
+											"source": "https://github.com/kevcunnane/azuredatastudio-newdatabase/releases/tag/0.0.2"
```

Comment:
just realized the version and the target doesn't match, is it intentional? I can correct it since i have a active pr out for this file

Reply:
Thanks for fixing - this was indeed a mistake.